host_core/test/e2e/echo_test.exs

```diff
@@ -111,15 +113,6 @@ defmodule HostCore.E2E.EchoTest do
         @httpserver_key
       )
 
-    # For now, okay to put a link definition without proper claims
```

Comment:
I was unsure why this needed to be called again, so if I miss-understood the test scenario let me know :).

Reply:
I think this was just a leftover copy/paste, good catch 😄